src/hyperlight_wasm/build.rs

```diff
@@ -124,21 +112,18 @@ fn build_wasm_runtime() -> PathBuf {
     let mut env_vars = env::vars().collect::<Vec<_>>();
     env_vars.retain(|(key, _)| !key.starts_with("CARGO_"));
 
-    let mut cargo_cmd = std::process::Command::new(&cargo_bin);
+    let mut cargo_cmd = cargo_hyperlight::cargo().unwrap();
     let mut cmd = cargo_cmd
         .arg("build")
+        .arg("--target-dir")
+        .arg(&target_dir)
         .arg("--profile")
         .arg(cargo_profile)
         .arg("-v")
-        .arg("--target-dir")
-        .arg(&target_dir)
         .current_dir(&in_repo_dir)
         .env_clear()
-        // On windows when `gdb` features is enabled this is not set correctly
-        .env("CFLAGS_x86_64_unknown_none", "-fPIC")
         .envs(env_vars)
-        .env("PATH", path_with(&toolchain_dir))
-        .env("HYPERLIGHT_GUEST_TOOLCHAIN_ROOT", &toolchain_dir);
+        .env("CFLAGS_x86_64_unknown_none", "-fPIC"); // Should this go on cargo hyperlight
```

Comment:
Isn't it added by cargo-hyperlight?

Reply:
I don't think it is. I took a note to ask and forgot to follow up.  @jprendes what do you think?

Reply:
https://github.com/hyperlight-dev/cargo-hyperlight/pull/7